Library/Homebrew/env_config.rb

```diff
@@ -318,12 +318,10 @@ module EnvConfig
                      "outdated.",
         boolean:     true,
       },
-      HOMEBREW_PIP_INDEX_URL:                    {
-        description: "If set, `brew install <formula>` will use this URL to download PyPI package resources.",
-      },
-      HOMEBREW_PIP_EXTRA_INDEX_URL:              {
-        description: "If set, `brew install <formula>` will also use this URL(s) to download PyPI package " \
-                     "resources in addition to `HOMEBREW_PIP_INDEX_URL`.",
+      HOMEBREW_PIP_INDEX_URLS:                   {
+        description:  "A space-separated list of URLs. If set, `brew install <formula>` will use these URLs to " \
```

Comment:
Wait, why does this need to be space separated? Why do you need multiple URLs here?

Reply:
For `pip`, the variable `PIP_INDEX_URL` should be a string of a single URL (1 URL), and the variable `PIP_EXTRA_INDEX_URL` can be a space-separated list (n URLs). If we concatenate these two variables, it would be a space-separated list (1 + n URLs).

Reply:
I think we should just have a single URL.